src/Libraries/Microsoft.Extensions.Http.Diagnostics/README.md

```diff
@@ -18,6 +18,99 @@ Or directly in the C# project file:
 </ItemGroup>
 ```
 
+## Usage Example
+
+### HTTP Client Logs Enrichment and Redaction
+
+These components enable enriching and redacting HTTP Client request logs. They remove built-it HTTP Client logging.
+
+In order to use the redaction feature, you need to reference the `Microsoft.Extensions.Compliance.Redaction` package.
```

Comment:
This is non-trivial and we need to improve discoverability of this (reference redaction package in `Http.Diagnostics` package?). I will create a work item to track that

Reply:
Created #4675

Reply:
Yeah I think we should reference the Redaction package automatically. Even when it is not used directly, the idea would be for folks referencing this package to be able to look at your sample code from the README and use it, but if we don't include the package reference they would get an error in the call to AddRedaction.

Reply:
That said, I agree with you we can fix this in a separate change.